en/docs/references/extend/rebranding/customizable-login-portal.md

```diff
@@ -1,12 +1,12 @@
-# Customize Login Pages
+# Customize Login Page Layouts
 
 You can customize the layout and branding of user interfaces like the login page, register page, username and password recovery pages, and single sign-on pages.
```

Comment:
```suggestion
You can customize the layout and branding of user interfaces like the login page, register page, username, and password recovery pages, and single sign-on pages.
```

Reply:
`username and password recovery pages` is one item